content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md

```diff
@@ -0,0 +1,165 @@
+  ([Issue #34855](https://github.com/istio/istio/issues/34855))
+
+- **Added** support for `trafficPolicy.loadBalancer.consistentHash` in `DestinationRule` for proxyless gRPC clients.
+  ([Pull Request #35333](https://github.com/istio/istio/pull/35333))
+
+- **Added** the ability for users to specify Envoy's `LOGICAL_DNS` as a connection type for a cluster using `DNS_ROUND_ROBIN` in ServiceEntry.
+  ([Issue #35475](https://github.com/istio/istio/issues/35475))
+
+- **Added** `failoverPriority` load balancing traffic policy, which allows users to set an ordered list of labels used to sort endpoints to do priority based load balancing.
+  ([Pull Request #34740](https://github.com/istio/istio/pull/34740))
+
+- **Added** support for creating mirrored QUIC listeners for non-passthrough HTTPS listeners at gateways
+  ([Pull Request #33817](https://github.com/istio/istio/pull/33817))
+
+- **Added** support for the `v1alpha2` version of the [gateway-api](https://gateway-api.org/).
+  ([Pull Request #35009](https://github.com/istio/istio/pull/35009))
+
+- **Added** experimental support for the `cluster.local` host behavior as defined by the Kubernetes Multi-Cluster Services (MCS) spec. This feature is off by default, but can be enabled by setting the following flags in Istio: `ENABLE_MCS_CLUSTER_LOCAL`, `ENABLE_MCS_HOST` and `ENABLE_MCS_SERVICE_DISCOVERY`. When enabled, requests to the `cluster.local` host will be routed to only those endpoints residing within the same cluster as the client.
+  ([Issue #35424](https://github.com/istio/istio/issues/35424))
+
+- **Fixed** Gateway API xRoute does not forward the traffic to that backend when weight `0`.
+  ([Issue #34129](https://github.com/istio/istio/issues/34129))
+
+- **Fixed** an issue in which ADS would hang due to the wrong `syncCh` size being provided.
+  ([Pull Request #34633](https://github.com/istio/istio/pull/34633))
+
+- **Fixed** and issue causing Ingress resources with the same name but different namespaces from conflicting.
+  ([Issue #31833](https://github.com/istio/istio/issues/31833))
+
+## Security
+
+- **Improved** performance of TLS certificate Secret watches to reduce memory usage.
+  ([Issue #35231](https://github.com/istio/istio/issues/35231))
+
+- **Added** support to istiod to notice `cacerts` file changes via the `AUTO_RELOAD_PLUGIN_CERTS` env var.
+  ([Issue #31522](https://github.com/istio/istio/issues/31522))
+
+- **Added** `VERTIFY_CERT_AT_CLIENT` environmental variable to istiod. Setting `VERTIFY_CERT_AT_CLIENT` to `true` will verify server certificates using the OS CA certificates when not using a `DestinationRule` `caCertificates` field.
+  ([Issue #33472](https://github.com/istio/istio/issues/33472))
+
+- **Added** Auto mTLS support for workload level peer authentication. You no longer need to configure destination rule when servers are configured with workload level peer authentication policy. This can be disabled by setting `ENABLE_AUTO_MTLS_CHECK_POLICIES` to "false".
```

Comment:
```suggestion
- **Added** Auto mTLS support for workload level peer authentication. You no longer need to configure a destination rule when servers are configured with workload level peer authentication policy. This can be disabled by setting `ENABLE_AUTO_MTLS_CHECK_POLICIES` to `false`.
```